aexpect/remote_door.py

```diff
@@ -616,6 +616,7 @@ def share_local_object(object_name, whitelist=None, host="localhost", port=9090)
     try:
         ns_server = Pyro4.locateNS(host=host, port=port)
         LOG.debug("Pyro4 name server already started")
+        ns_daemon = None
```

Comment:
It would be better to initialize the `ns_daemon` outside the `try/except` block. I know the except block currently results into `nsd_running == False` so the `ns_daemon` would not be checked, but if the code changes somehow everything could break.

Reply:
Sounds good, just wanted to minimize the changes and still make the linters happy.